scripts/release_scripts/gcloud_adapter.py

```diff
@@ -23,7 +23,7 @@
 import sys
 
 GCLOUD_PATH = os.path.join(
-    '..', 'oppia_tools', 'google-cloud-sdk-251.0.0', 'google-cloud-sdk',
+    '..', 'oppia_tools', 'google-cloud-sdk-304.0.0', 'google-cloud-sdk',
     'bin', 'gcloud')
```

Comment:
Should this be moved to some commonplace (it is also defined in _restore_backup.py_)? Maybe _release_constants.py_

Reply:
Done.